lib/shopify_transporter/exporters/magento/product_exporter.rb

```diff
@@ -65,6 +67,12 @@ def info_for(product_id)
             .call(:catalog_product_info, product_id: product_id)
             .body[:catalog_product_info_response][:info]
         end
+
+        def image_attribute(product_id)
```

Comment:
nit: shouldn't this be `image_attributes` or `images_attribute` or something since it's an array?